Trustroots/trustroots
# Update/donate page

#### Proposed Changes

* Change the text and the layout of the `/donate` page (Fixes Trustroots/community#53)
* Rename the `/donate` page to `/contribute` (Fixes Trustroots/community#50)
* Rename 'Donate' -> 'Contribute' in footer and header (Fixes Trustroots/community#49)
* Remove `/donate/help` and `/donate/policy` pages (Fixes Trustroots/community#51)

#### Testing Instructions

- Check the text and the layout of the `/contribute` page
- Check that links that used to lead to `/donate` page now lead to `/contribute` page
- Check 'Donate' is renamed to 'Contribute' in footer and header
- Check that there are no links that lead to `/donate/help` and `/donate/policy` pages
